scripts/prepare_release.sh

```diff
@@ -78,10 +79,18 @@ mvn versions:set versions:commit -DnewVersion=${NEXT_SNAPSHOT} -DgenerateBackupP
 # Commits this next snapshot version.
 git commit -am "${NEXT_SNAPSHOT}"
 
-# Pushes the tag and release branch to Github.
+# Push the tag and release branch to Github.
 git push origin v${VERSION}-${SUFFIX}
 git push --set-upstream origin ${VERSION}-${SUFFIX}
 
+# Create the PR
+gh pr create -t "Release ${VERSION}-${SUFFIX}" -b "Release ${VERSION}-${SUFFIX}"
+
 # File a PR on Github for the new branch. Have someone LGTM it, which gives you permission to continue.
-EchoGreen 'File a PR for the new release branch:'
+EchoGreen 'Ask someone to approve this PR:'
 echo https://github.com/GoogleCloudPlatform/cloud-opensource-java/compare/${VERSION}-${SUFFIX}
```

Comment:
This URL seems to compare a branch with master.

Reply:
For example https://github.com/GoogleCloudPlatform/cloud-opensource-java/compare/auto shows "View Pull Request" button. So it should be fine.